moon-store-mac-os/moon-store-mac-os/Helpers/AlertManager/AlertManager.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  AlertManager.swift
+//  moon-store-mac-os
+//
+//  Created by Diana Zeledon on 4/12/24.
+//
+
+import AppKit
+
+final class AlertManager: ObservableObject {
+    @Published var isPresented: Bool = false
+    
+    func showAlert(type: AlertType) {
+        isPresented = true
+        alertConstruction(type)
+    }
+    
+    private func alertConstruction(_ alertType: AlertType) {
+        let alert = NSAlert()
+        alert.alertStyle = .informational
+        alert.messageText = alertType.title
+        alert.informativeText = alertType.message
+
+        if let icon = alertType.icon {
```

Comment:
Why this icon is optional?

Reply:
Because `NSImage` is optional, and if for some reason you can't find the image don't add it or should a default image be added?

Reply:
The best way to use is using only image name

Reply:
The responsibility for build and images is from a View

Reply:
Ok, will pass only the name